apps/www/package.json

```
@@ -11,7 +11,7 @@
     "preview": "vitepress preview",
     "typecheck": "vue-tsc --noEmit",
     "typecheck:registry": "vue-tsc --noEmit -p tsconfig.registry.json",
-    "build:registry": "typecheck:registry && tsx ./scripts/build-registry.ts"
+    "build:registry": "npm run typecheck:registry && tsx ./scripts/build-registry.ts"
```

I forgot to add `pnpm`, change `npm run` to `pnpm` please

```suggestion
    "build:registry": "pnpm typecheck:registry && tsx ./scripts/build-registry.ts"
```